optimize_versions.json

```diff
@@ -1 +1 @@
-["3.12.0", "3.11.0", "3.10.0", "3.9.0", "3.7.0"]
+["3.13.0", "3.12.0", "3.11.0", "3.10.0", "3.9.0", "3.7.0"]
```

Comment:
3.9.0 gets unsupported & archived, ya? So it shouldn't be on this list...?

Reply:
It gets unsupported and archived, yes. 

But I can't remove it from these docs until I go through the process of archiving it.

Reply:
I have "Archive 8.1/3.9.0" as a task in the non-blocking follow-up tasks on #3250 (no issue created for it yet). When _that_ is complete, it would disappear from this list.

Reply:
Thank you! I couldn't remember if that got removed here or later. Will resolve!

Reply:
Also, in https://github.com/camunda/camunda-docs/pull/3630/commits/8b5c57bdffd4cf49a2694e6c08d0f0314235c47a, the banners are updated so that v8.1/3.9.0 now have "unsupported" banners. These will remain as indicators of lack of support, until they are archived.